deployments/arbitrum-goerli/usdc/configuration.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "Compound USDC",
+  "symbol": "cUSDCv3",
+  "baseToken": "USDC",
+  "baseTokenAddress": "0x72A9c57cD5E2Ff20450e409cF6A542f1E6c710fc",
```

Comment:
Note: Need to double-check that these are the canonical bridged tokens from Goerli.

Reply:
I updated the asset addresses in #719